applications/FluidDynamicsApplication/custom_utilities/fluid_element_data.cpp

```diff
@@ -88,6 +88,19 @@ void FluidElementData<TDim, TNumNodes, TElementIntegratesInTime>::FillFromHistor
     }
 }
 
+template <size_t TDim, size_t TNumNodes, bool TElementIntegratesInTime>
+void FluidElementData<TDim, TNumNodes, TElementIntegratesInTime>::FillFromHistoricalNodalData(
+    NodalTensorData& rData,
+    const Variable<Matrix>& rVariable,
+    const Geometry<Node<3>>& rGeometry)
+{
+    for (size_t i = 0; i < TNumNodes; i++) {
+        const Matrix& r_nodal_values =
+            rGeometry[i].FastGetSolutionStepValue(rVariable);
+        rData[i] = r_nodal_values;
+    }
```

Comment:
```suggestion
    for (size_t i = 0; i < TNumNodes; ++i) {
        rData[i] = rGeometry[i].FastGetSolutionStepValue(rVariable);
    }
```
To avoid the auxiliary reference and the iterator copy in `++i`

Reply:
I have written i++ to be consistent with the rest of `FillFromHistoricalNodalData` defined in the element_data

Reply:
OK. Then up to you.